### ISSUE TITLE
apk add nkf

### ISSUE THREAD

**Issue by blakelapierre** [tom-tan/alpine-pkg-nkf#1]
Running `apk add nkf` doesn't work for me. Is nfk in the default Alpine 'repository'?

**Comment by tom-tan** [comment]
> Is nfk in the default Alpine 'repository'?

No. You have to download and install `nkf-2.1.4-r0.apk` manually or setup the repository by yourself.
